test/clojure/nrepl/sanity_test.clj

```diff
@@ -70,6 +70,18 @@
     ["" 4 20 1 0] "(+ 2 2) (* *1 5) (/ *2 4) (- *3 4)"
     ["" nil] '*1))
 
+(deftest read-error-short-curcuits-execution
```

Comment:
I think you using a couple of `context`s will make this test a bit better.

Reply:
What's a context?

Reply:
Sorry, too much Ruby - I meant https://clojuredocs.org/clojure.test/testing. It's `context` in RSpec. :-)